docs/reference/platforms/unix-prerequisites.rst

```diff
@@ -5,31 +5,37 @@ some of the common alternatives:
     The package list should be the same as in ci.yml, and the BeeWare tutorial
     (CI will also have WebView requirements)
 
-**Ubuntu 18.04+ / Debian 11+**
+**Ubuntu / Debian**
 
 .. code-block:: console
 
     (venv) $ sudo apt update
-    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
+    (venv) $ sudo apt install gcc git pkg-config python3-dev gir1.2-gtk-3.0 libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
```

Comment:
We use `build-essential` instead of `gcc` in the Dockerfile for briefcase-linux-system-template... any reason not to do the same here?

Reply:
I was trying to be more conservative because we know the dependencies for the packages we're trying to install. I'm mostly ambivalent, though, so I can use `build-essential` (or equivalent).

Reply:
If we were talking about installing an esoteric group of packages, I'd probably agree - but this is basic build dependencies like gcc and make - I don't think there's anything to extravagant in that list.

Reply:
Fedora does have `yum groupinstall "Development Tools"`....but it installs _so much_...